File: app/src/main/java/com/tomatogaming/mealmelt/Recipe.java
package com.tomatogaming.mealmelt;

import android.graphics.drawable.Drawable;

import java.util.List;

public class Recipe
{
    protected int Id;
    protected String Name;
    protected String Description;
    protected Author Author;
    protected Category Category;
    protected List<Recipe> Sides;
    protected List<Step> Steps;
    protected List<Ingredient> Ingredients;
    protected List<Tool> Tools;
    protected Drawable Photo;
}

File: EFMigrations/Recipe.cs
using System.ComponentModel.DataAnnotations;

namespace MealMelt.Repository.Models
{
    public class Recipe
    {
        [Key]
        public int Id { get; set; }
        public string Name { get; set; }
        public string Author { get; set; }
        public Category Category { get; set; }
        public int? PhotoId { get; set; }
    }
}